### ISSUE TITLE
Comments between package name and imports cause `panic: New node is nil`

### ISSUE THREAD

**Issue by pauln** [metaverse/truss#202]
This one had us scratching our heads for a while!

If you have any comments between the `package` line and the first `import`, Truss fails with the following error:
```
panic: New node is nil, namepath: '[echo.proto github.com/TuneLab/truss/deftree/googlethirdparty/annotations.proto]' cur_node: '&{<nil> echo.proto  [0xc04239d270 0xc04239d2c0 0xc04239d310] [] [0xc042034480]}'


goroutine 1 [running]:
github.com/TuneLab/truss/deftree.(*MicroserviceDefinition).SetComment(0xc0422dfcc0, 0xc04224f380, 0x2, 0x2, 0xc04235fbe8, 0x4)
        C:/Users/Paul/go/src/github.com/TuneLab/truss/deftree/deftree.go:151 +0x1f7
github.com/TuneLab/truss/deftree.AssociateComments(0x8c7260, 0xc0422dfcc0, 0xc0422de190)
        C:/Users/Paul/go/src/github.com/TuneLab/truss/deftree/associate_comments.go:269 +0x944
github.com/TuneLab/truss/deftree.New(0xc0422de190, 0x8c1e60, 0xc042004720, 0xc042004720, 0x0, 0x0, 0xc0422de190)
        C:/Users/Paul/go/src/github.com/TuneLab/truss/deftree/build_deftree.go:72 +0x509
main.parseServiceDefinition(0xc042050780, 0x0, 0x744b67, 0x12, 0x0, 0x0)
        C:/Users/Paul/go/src/github.com/TuneLab/truss/cmd/truss/main.go:283 +0x5a7
main.main()
        C:/Users/Paul/go/src/github.com/TuneLab/truss/cmd/truss/main.go:104 +0x126
```

Removing the offending comment(s) allows Truss to operate correctly.

An example of troublesome comments:
```
syntax = "proto3";
package echo;

// This comment will cause Truss to panic.
import "github.com/TuneLab/truss/deftree/googlethirdparty/annotations.proto";
```

We encountered this issue as we had comments separating our imports into `// External imports` and `// Internal imports`, but it seems that any comment at all before the first import will trigger this issue.

**Comment by hasLeland** [comment]
@pauln thank you for the bug report! I can't give an exact timeline on this bug, but we've added it to the team sprint and I'll be digging into fixing this.

From a cursory look at the code, it seems that `truss` is written with a bug in it such that if you attempt to associate a comment with something that this portion of the codebase doesn't care about or keep track of, then truss incorrectly panics instead of returning an error which can be handled.

IMO, the fix is to change [this function](https://github.com/TuneLab/truss/blob/096ba69249ea9c36208500413b665307c7431d00/deftree/deftree.go#L151) so that it returns an error which can be logged/handled/ignored as needed.

**Comment by hasLeland** [comment]
@pauln I actually had some additional time today, so I put together a PR for this issue a bit ahead of schedule: https://github.com/TuneLab/truss/pull/206

**Comment by pauln** [comment]
This has been fixed by #206.
